zhmcclient/_partition.py

```diff
@@ -773,3 +773,110 @@ def wait_for_status(self, status, status_timeout=None):
                     actual_status, statuses, status_timeout)
 
             time.sleep(1)  # Avoid hot spin loop
+
+    @logged_api_call
+    def increase_crypto_config(self, crypto_adapters,
+                               crypto_domain_configurations):
+        """
+        Adds the specified adapters and/or domain configurations
+        to the crypto configuration of the corresponding partition.
+
+        Authorization requirements:
+
+        * Object-access permission to this Partition.
+        * Task permission to the "Partition Details" task.
+
+        Parameters:
+
+          crypto_adapters (list): Crypto adapters that should be added
```

Comment:
1. The type 'list' creates an incorrect link (to LparManager.list()).
2. Specify the Python types of the list items.